index.html

```diff
@@ -1317,6 +1361,34 @@
     <a>Web NFC message</a>s either from an <a>NFC tag</a> or an <a>NFC peer</a>.
   </p>
 
+  <section> <h3>The <strong>NFCPushTarget</strong> enum</h3>
+    <dl title="enum NFCPushTarget" class="idl">
+      <dt>tag</dt>
+      <dt>peer</dt>
+      <dt>any</dt>
+    </dl>
+  </section>
+
+  <section> <h3>The <strong>NFCPushOptions</strong> dictionary</h3>
+    <dl title="dictionary NFCPushOptions" class="idl">
```

Comment:
I used to use contiguous IDL, until at some point in SysApps we started using the dl style. 
Anyway already in this PR I'd like to also include the full IDL as an Appendix (as contiguous IDL).

I will look into Bikeshed after this PR is landed, I hope (and likely) it's better than Anolis. Otherwise I used to prefer ReSpec - I like writing the page in an editor configured with autocomplete etc, and I like that I don't have to generate index.html every time I want to see the changes.